pizza-facts-&-trivia.md

```diff
@@ -23,6 +23,7 @@ We've categorized the facts into different sections to help explore the diverse
 - An International Pizza Expo takes place each year in Las Vegas, Nevada. It’s the world’s largest gathering of members from the pizza industry. Unfortunately, it’s not open to the public. You have to be in the pizza business to attend.
 - As of 2017, Domino’s Pizza is the most popular pizza chain in the world. The company has more than 17,000 pizza stores in 90 countries, and this number continues to grow.
 - Anchovies are America's least favorite topping.
+- The longest pizza ever made was over 1,930.39 meters (6,333 ft 3.60 in) feet long. This record-breaking pizza was created in Fontana, California, USA. The pizza was made using approximately 17,700 lbs of dough, 5,000 lbs of tomato sauce and 3,900 lbs of mozzarella cheese.
```

Comment:
@adiati98 can you please explain the reason behind the changes you suggested

Reply:
@Ezzywealth there are a few points here:

- "The longest pizza ever made was over 1,930.39 **meters** (6,333 ft 3.60 in) **feet long**." --> You don't need to say "feet long" after meters . And "m" for meters is more common to use than "meters".
- A food is "made" or "cooked" and not "created".
- Whenever we can, we want to avoid repetition. The word "made" is used  in the first and the third sentence and both are refering to the pizza making.
- As this is a "record-breaking" pizza (does it mean guiness book record?), it would be great to also mention the fact when it was achieved. 

I hope these make sense. :)